src/core/components/LandingPage/Features/FeatureCard.tsx

```diff
@@ -20,7 +20,16 @@ const FeatureCard = ({ card }: { card: FeatureCardType }) => {
   return (
     <div css={cssStyles.card}>
       <DecorationPlus>
-        {badge && <Badge content={badge} styles={cssStyles.badge} bgColor={theme.colors.black[22]} />}
+        {badge && (
+          <Badge
+            content={badge}
+            borderStyles={cssStyles.badge}
+            styles={{
+              padding: `${convertPxToRem(6.2)} ${theme.spacing.medium}`,
```

Comment:
maybe it's better to ask designer to update mock instead of using 6.2px as a padding?